sdk-lib/src/main/java/cash/z/ecc/android/sdk/jni/RustBackend.kt

```diff
@@ -227,7 +227,8 @@ internal class RustBackend private constructor(
             memo ?: ByteArray(0),
             File(saplingParamDir, SaplingParamTool.SPEND_PARAM_FILE_NAME).absolutePath,
             File(saplingParamDir, SaplingParamTool.OUTPUT_PARAM_FILE_NAME).absolutePath,
-            networkId = network.id
+            networkId = network.id,
+            useZip317Fees = false
```

Comment:
I suggest we eliminate the default argument here, define a constant like below, and pass the constant in explicitly where this is called.

I often dislike default arguments because they are a frequent source of subtle bugs.

```
companion object {
    private const val IS_USE_ZIP_317_FEES = false
}
```

Reply:
This isn't a default argument? This is a function call of the `createToAddress` FFI method, passing the explicit value `false`. It only uses keyword argument styling because that's what was already done for `networkId`, and I'm bad at Kotlin and don't know how positional and keyword arguments interact. (An earlier version of the PR did have default arguments elsewhere higher up in the stack, but all those were removed when you requested that SDK users not be allowed to configure this.) Happy to use a constant here though.